src/main/java/com/cascadebot/cascadebot/commands/developer/GuildSaveSubCommand.java

```diff
@@ -0,0 +1,27 @@
+package com.cascadebot.cascadebot.commands.developer;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.ICommandExecutable;
+import com.cascadebot.cascadebot.data.mapping.GuildDataMapper;
+import com.cascadebot.cascadebot.permissions.CascadePermission;
+import net.dv8tion.jda.core.entities.Member;
+
+public class GuildSaveSubCommand implements ICommandExecutable {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        GuildDataMapper.getGuilds().invalidate(context.getGuild().getIdLong());
```

Comment:
This should probably be default only save the current guild, but accept guild id and `all`

Reply:
+1 for this